### PR TITLE
String Tokenizer

### PR DESCRIPTION
**Pull request by AniketKumar2500**
I updated tokenize function to tokenize strings as well as indents.

### CODE REVIEW

File: Tokenizer/tokenizer.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 '''
 Python implementation of Csq Tokenizer
 '''
-from ..Grammar.grammer import *
+from Grammar.grammar import *
```
Comment (WithSJ) [inline Tokenizer/tokenizer.py:4]: from ..Grammar.grammer import * 
This import statement uses relative import to access a module that is in a parent directory of the current module.
and i think we should use this

Comment (AniketKumar2500) [reply Tokenizer/tokenizer.py:4]: The code is called by main.py file due to which relative import is not required and the 2nd  thing is that it's very much prone to errors i tried to use it but due to some sort of file sturct error relative import was blocked.

**Comment by AniketKumar2500** [comment]
I knew it but when I run it I got errors.

File: Tokenizer/tokenizer.py (hunk 1 above)
Comment (WithSJ) [reply Tokenizer/tokenizer.py:4]: okye.

**Review by WithSJ — APPROVED** [review]

**Pull request merged**